### PR TITLE
#51: Status page should be consistent between WiFi and Ethernet

### PR DESCRIPTION
**Pull request by TheSomeMan**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -579,55 +561,79 @@ <h1>
...
+                    <span lang="en">Ruuvi Gateway is now ready to be used.</span>
+                    <span lang="fi">Ruuvi Gateway on nyt valmiina käytettäväksi.</span>
+                </h2>
+
                 <div class="ingress">
-                    <p lang="en">Your Ruuvi Gateway is now connected! If you want to modify the settings, press and hold
-                        CONFIGURE button on bottom of the Gateway for at least five seconds.</p>
-                    <p lang="fi">Ruuvi Gateway on nyt toiminnassa! Jos haluat muokata asetuksia, paina ja pidä pohjassa
-                        vähintään viisi sekuntia CONFIGURE-näppäintä gatewayn takapuolella.</p>
+                    <p lang="en">If you want to modify the settings, disconnect the Ethernet cable from the Gateway,
```
Comment (TheSomeMan) [inline src/index.html:586]: @ojousima could you check this translation into Finnish?

File: src/index.html [hunk 2]
```diff
@@ -579,55 +561,79 @@ <h1>
...
-                    <span lang="fi">Yhdistetty verkkoon </span>
-                    "<span class="wifi-network-name"></span>".
-                </p>
-                <p class="connect-cable hidden">
-                    <span lang="en">Connected to internet via Ethernet</span>
-                    <span lang="fi">Yhdistetty internetiin Ethernet-yhteydellä</span>
+                <p>
+                    <div id="connected-wifi">
+                        <span lang="en">Connected to Wi-Fi: </span>
+                        <span lang="fi">Yhdistetty verkkoon WLAN-verkkoon: </span>
```
Comment (TheSomeMan) [inline src/index.html:595]: @ojousima could you check this translation into Finnish?

File: src/index.html [hunk 3]
```diff
@@ -579,55 +561,79 @@ <h1>
...
+                        "<span class="wifi-network-name"></span>".
+                    </div>
+                    <div id="connected-eth" class="hidden">
+                        <span lang="en">Connected to Ethernet</span>
+                        <span lang="fi">Yhdistetty Ethernetiin</span>
```
Comment (TheSomeMan) [inline src/index.html:600]: @ojousima could you check this translation into Finnish?

File: src/index.html [hunk 4]
```diff
@@ -639,94 +645,42 @@ <h2>
...
             </div>
+
+            <section>
+                <div id="diag-disconnect" class="diag-box">
+                    <div class="diag-box-win">
+                        <table>
+                            <tr>
+                                <td colspan="2">
+                                    <span lang="en">Are you sure you would like to disconnect from the network?</span>
+                                    <span lang="fi">Haluatko varmasti katkaista yhteyden tähän verkkoon?</span>
```
Comment (TheSomeMan) [inline src/index.html:659]: @ojousima could you check this translation into Finnish?

File: src/index.html (hunk 1 above)
Comment (ojousima) [reply src/index.html:586]: If you want to modify the settings, disconnect the Ethernet cable from the Gateway
-> "Jos haluat muokata asetuksia, irroita Ethernet-kaapeli Gatewaystä,"

File: src/index.html (hunk 2 above)
Comment (ojousima) [reply src/index.html:595]: "Yhdistetty WLAN-verkkoon:", remove first "verkkoon"

File: src/index.html (hunk 3 above)
Comment (ojousima) [reply src/index.html:600]: ok

File: src/index.html (hunk 4 above)
Comment (ojousima) [reply src/index.html:659]: ok

**Review by ojousima — APPROVED** [review]
Thanks, please check the comments on translations

**Pull request merged**